.github/workflows/automatus-ubuntu2404.yml

```diff
@@ -5,6 +5,8 @@ on:
 concurrency:
   group: ${{ github.workflow }}-${{ github.event.number || github.run_id }}
   cancel-in-progress: true
+env:
+  DATASTREAM: ssg-ubuntu2404-ds.xml
```

Comment:
Nice catch, thanks!
I suggest we move the fix to the workflow to a separate PR though.